### PR TITLE
styles(Tabs): updates accordion styles when it gets rendered inside tabs

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Update TabList width
* Update spacing between tags and accordion items: 4px

## Justify why they are needed

Requested by Zak

image

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1807** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1807" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1807?utm_source=stack-comment).

File: apps/store/src/blocks/AccordionBlock.tsx [hunk 1]
```diff
@@ -17,42 +17,30 @@ type Props = SbBaseBlockProps<{
...
+        {displayTitleDescriptionSection && <AccodrionTitleDescription blok={blok} />}
+        <StyledAccordionRoot type="single" collapsible>
+          {accordionItems.map((nestedBlock) => (
+            <AccordionItemBlock key={nestedBlock._uid} blok={nestedBlock} />
+          ))}
+        </StyledAccordionRoot>
+      </>
+    )
+  } else {
+    content = (
```
Comment (gustaveen) [inline apps/store/src/blocks/AccordionBlock.tsx:39]: nit: Could we use the `ConditionalWrapper` instead :)

File: apps/store/src/blocks/TabsBlock.tsx [hunk 2]
```diff
@@ -77,6 +80,21 @@ const Title = styled(Heading)({
   },
 })
 
+const TabLisWrapper = styled(GridLayout.Content)({
```
Comment (gustaveen) [inline apps/store/src/blocks/TabsBlock.tsx:83]:
```suggestion
const TabListWrapper = styled(GridLayout.Content)({
```

**Review by gustaveen — COMMENTED** [review]
Nice! Just two comments 🙌

File: apps/store/src/blocks/AccordionBlock.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/blocks/AccordionBlock.tsx:39]: The issue is that the dom changes drastically on the two scenarios; I mean, we're not just adding a wrapper when that block is not nested. We also add a grid structure and so on.

**Review by gustaveen — APPROVED** [review]
Awesome! 🙌

File: apps/store/src/blocks/AccordionBlock.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/blocks/AccordionBlock.tsx:39]: Gotcha!

**Pull request merged**